back/src/adapters/primary/authMiddleware.ts

```diff
@@ -281,3 +281,65 @@ export const verifyJwtConfig = <K extends JwtKind>(config: AppConfig) => {
 
   return { verifyJwt, verifyDeprecatedJwt };
 };
+
+const responseErrorForV2 = (res: Response, message: string, status = 403) =>
+  res.status(status).json({ message: `forbidden: ${message}`, status });
```

Comment:
je te propose de virer le `forbidden`:
```suggestion
  res.status(status).json({ message, status });
```